scripts/build_example

```diff
@@ -41,28 +41,47 @@ else
     TARGET="$NAME"
 fi
 
+bazel_build_flags+=(
+    '--symlink_prefix=bazel-client-'
+    "--compilation_mode=$COMPILATION_MODE"
+)
+
 readonly RUST_FOLDER="$(find examples/"$TARGET" -type d -name rust)"
 if [[ -n "$RUST_FOLDER" ]]; then
     for module in examples/"$TARGET"/module*/rust/Cargo.toml; do
-    cargo build --release --target=wasm32-unknown-unknown --manifest-path="$module"
+        cargo build --release --target=wasm32-unknown-unknown --manifest-path="$module"
     done
-fi
 
-bazel_build_flags+=(
-    '--symlink_prefix=bazel-client-'
-    "--compilation_mode=$COMPILATION_MODE"
-)
+    # Serialize application configuration for Rust module.
+    if [[ "hello_world" == "$TARGET" ]]; then
+        # `hello_world` example has an optional additional `translator` module.
+        cargo build --release --target=wasm32-unknown-unknown --manifest-path="examples/translator/module/rust/Cargo.toml"
 
-# Use a different output_base so that we don't lose incremental state.
-# See https://docs.bazel.build/versions/master/command-line-reference.html#flag--output_base.
-bazel --output_base="$CACHE_DIR/client" build "${bazel_build_flags[@]}" "//examples/$TARGET/client"
+        bazel --output_base="$CACHE_DIR/client" build "${bazel_build_flags[@]}" \
+            "//examples/$TARGET/config:config" \
+            "//examples/$TARGET/config:config_translator"
```

Comment:
Maybe it's clearer to replace `$TARGET` with `hello_world` here, since it's very specific to that already.